yetanotherco/aligned_layer
# chore: bump Sp1 to v1.0.8-testnet

# Changes
SP1 updated to https://github.com/succinctlabs/sp1/releases/tag/v1.0.8-testnet

# Important
User should download SP1 proof again
`curl -L https://raw.githubusercontent.com/yetanotherco/aligned_layer/main/batcher/aligned/get_proof_test_files.sh | bash`

# How to test
0. `make build_sp1_macos`
1. `make anvil_start_with_block_time`
2. `make aggregator_start`
3. Run the operator
```
export OPERATOR_ADDRESS=0x70997970C51812dc3A010C7d01b50e0d17dc79C8
make operator_full_registration CONFIG_FILE=config-files/config-operator-1.yaml
make operator_start CONFIG_FILE=config-files/config-operator-1.yaml
```
4. `make batcher_start`
5. Send at least 2 SP1 proofs
```
make batcher_send_sp1_task
```
Everything should work fine
```shell
[2024-07-02T17:41:43Z WARN  aligned] Missing keystore used for payment. This proof will not be included if sent to Eth Mainnet
[2024-07-02T17:41:43Z INFO  aligned] Submitting proofs to the Aligned batcher...
[2024-07-02T17:42:19Z INFO  aligned] Batch inclusion data written into ./aligned_verification_data/b0ef218f_0.json
[2024-07-02T17:42:19Z INFO  aligned] Proofs submitted to aligned. See the batch in the explorer:
[2024-07-02T17:42:19Z INFO  aligned] https://explorer.alignedlayer.com/batches/0xb0ef218f57ccd5f1e838aea55d79d9200dd4e6b6edf2a5cb55b1f422b80a9c74
```
